src/OrchardCore.Modules/OrchardCore.ReCaptcha/Drivers/ReCaptchaLoginFormDisplayDriver.cs

```diff
@@ -1,34 +0,0 @@
-using OrchardCore.DisplayManagement.Handlers;
-using OrchardCore.DisplayManagement.Views;
-using OrchardCore.ReCaptcha.Configuration;
-using OrchardCore.ReCaptcha.Services;
-using OrchardCore.Settings;
-using OrchardCore.Users.Models;
-
-namespace OrchardCore.ReCaptcha.Drivers;
-
-public sealed class ReCaptchaLoginFormDisplayDriver : DisplayDriver<LoginForm>
```

Comment:
You want a driver here to be able to extend the LoginForm with other things.

Reply:
@sebastienros 
No longer needed. We are rendering this using ShapeTableProvider. Look at the ReCaptchaShapeTableProvider class in this PR

Reply:
I understand, but I think this should be driven by a driver. Or this is not extensible anymore.
What if you want another component extending the login form with another form or validation.

Reply:
In order to fix linked bug, we have to remove it out of the DisplayDriver. `LoginForm` is still extensible. But we can't render the recaptcha from there as it screwes the robot counter.

Reply:
If at least there was a way to take placement into account in this provider. Otherwise this is breaking. It seems like the shape from the driver was called "FormReCaptcha"

Reply:
the driver was returning a shape result using the `FormReCaptcha` type.  I think you can still use placement to targeting `LoginForm_Edit`. Maybe we should specify a differentiator.